julia/core.py

```diff
@@ -286,7 +289,11 @@ def __init__(self, init_julia=True, jl_init_path=None):
                     "Install PyCall by running the following line:\n"
                     """\tjulia -e 'Pkg.add("PyCall"); Pkg.update()'\n""")
             try:
-                self.call('pyinitialize(C_NULL)')
+                if os.name == "nt":
+                    # on windows C_NULL doesn't mean anything
+                    self.call('pyinitialize()')
```

Comment:
So, if my patch allows to have an "extremely fragile" pyjulia on windows, instead of an "already broken" one, shouldn't you call that "progress" and merge it ?